.goreleaser.yml

```diff
@@ -16,9 +17,18 @@ builds:
   ldflags:
     - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
   goos:
+    - freebsd
+    - windows
     - linux
+    - darwin
   goarch:
     - amd64
+    - '386'
+    - arm
+    - arm64
+  ignore:
+    - goos: darwin
+      goarch: '386'
```

Comment:
These changes should all be reverted as we're only building linux amd64.

Reply:
removed.